spec/controllers/approval_requests_controller_spec.rb

```diff
@@ -0,0 +1,346 @@
+require 'rails_helper'
+
+RSpec.describe ApprovalRequestsController do
```

Comment:
This is the default behavior of RSpec generator.
I don't mind to leave it as it is.
Also, minitest is able to mimic RSpec DSL. Proof: 
```
require "minitest/autorun"

describe Meme do
  before do
    @meme = Meme.new
  end

  describe "when asked about cheeseburgers" do
    it "must respond positively" do
      @meme.i_can_has_cheezburger?.must_equal "OHAI!"
    end
  end

  describe "when asked about blending possibilities" do
    it "won't say no" do
      @meme.will_it_blend?.wont_match /^no/i
    end
  end
end
```
https://github.com/seattlerb/minitest

Other interesting facts can be found [here](http://stackoverflow.com/questions/24747448/rspec-describe-vs-describe)